careers/software-engineer.md

```diff
@@ -4,52 +4,56 @@ The goal of the Software Engineer path is to train engineers to master the funda
 
-**Communication:** They communicate well and are capable of delivering feedback to peers and their manager. When given a task with unclear requirements they know how to ask for clarification, and ensure that all assumptions are vetted before work starts to reduce the need for re-work. They understand how their work fits into the larger picture for their team, and use this to identify conflicting requirements to their people leader and product manager. As Engineers learn they will start to participate more in the technical design process, often with guidance from senior engineers. Consistently speaks up in Sprint Planning meetings to ensure complete understanding of each work item. An Engineer II actively seeks out knowledge to supplement the team's weak points and areas of interest, including attending meetups, reading books, etc.
+**Impact:** They can take the lead on delivering work critical to meeting the sprint goal and are capable of independently planning work that multiple engineers can complete in a sprint or two. Instead of just executing on a task as written, they think critically about the problem and ensure that the solution will thoroughly solve the problem.
 
-Engineer IIs can take a bug or a task, and instead of just executing on the task as written, think critically about the problem and ensure that the issue has been thoroughly resolved by identifying root causes, proactively addressing performance concerns, providing proactive feedback on any product requirements and writing solid tests.
+**Communication:** They are able to effectively review code in their area of expertise and regularly point out opportunities for improvement in the code that they review. They proactively point out concerns with proposed or current sprint work as soon as they see them. They can articulate the tradeoffs of different approaches to solving a problem.
 
-By the time an engineer is ready to be promoted to Engineer III they will have focused on a technology or functional area as their expertise and become capable of mentoring interns and new engineers in these areas.
+**Collaboration:** They are mindful of the work that others are doing and can critique or suggest improvements to the proposed plans of others. They get help from others when stuck, but are also capable of helping unstick their peers. They are capable of effectively running small to medium-sized projects with regular coaching.
 
 ## Engineer III
 
-**Technical Ability:** An Engineer III can consistently be relied on to take on an undefined project through to execution while fully and effectively utilizing the skills and intelligence of the team. An Engineer III has the ability to provide an answer to most technical questions that the team may have, but they make a choice to exercise restraint and ask questions to help other team members reach their own conclusions.
+An engineer III is capable of taking on complex technical problems and coordinating work spanning several months.
```

Comment:
The usage of time-scales is probably OK here. But, what we really care about is which Engineers are capable of leading projects. Yet, saying "DRIs projects" is probably a bit harder to make clear since they came in all sizes and shapes.

Based on how this is written, I assume E III's are the when folks can start DRI'ing things (for the most part).

Reply:
We don't have hard and fast rules around this, but generally speaking, yes. E3+ is when we typically have people start DRIing large projects. And most of our projects are large.